pom.xml

```diff
@@ -332,25 +331,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.2</version>
+        <version>3.1.0</version>
```

Comment:
Latest is `3.1.1` released in December.

https://search.maven.org/search?q=g:org.apache.maven.plugins%20AND%20a:maven-assembly-plugin&core=gav

Reply:
Should be `3.1.1` then.